app/policies/casa_case_policy.rb

```diff
@@ -89,35 +88,44 @@ def permitted_attributes
   end
 
   def same_org_supervisor_admin_or_assigned?
-    is_in_same_org? && (
-    admin_or_supervisor? || is_volunteer_actively_assigned_to_case?
-  )
+    admin_or_supervisor_same_org? || is_volunteer_actively_assigned_to_case?
   end
 
   def same_org_supervisor_admin?
-    is_in_same_org? && admin_or_supervisor?
+    admin_or_supervisor_same_org?
   end
 
   def index?
-    admin_or_supervisor_or_volunteer?
+    admin_or_supervisor_or_volunteer_same_org?
   end
 
   alias_method :show?, :same_org_supervisor_admin_or_assigned?
-  alias_method :save_emancipation?, :index?
+  alias_method :save_emancipation?, :index? # Should this be the same as edit?
```

Comment:
I think only @compwron will know the answer to this (and if not, we should just approve it)